app/main.py

```diff
@@ -1,3 +1,52 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)):
+        self.km = km
+
+    def __str__(self) -> str:
+        return f'Distance: {self.km} kilometers.'
+
+    def __repr__(self) -> str:
+        return f'Distance(km={self.km})'
+
+    def __add__(self, other):
```

Comment:
add type annotations everywhere

Reply:
yes, please, add type annotation for:
1) `other` variable
2) function. If function returns nothing then `-> None`